fixed broken links, moved images and other minor fixes

mairaw opened the pull request:
Fixing a number of broken links present in the build report. Since there were some links to images that were broken, I took this opportunity to also move them to the right place according to docs.microsoft.com guidelines for consistency. I also found some typos along the way, so I fixed those too.

We still have some art files under core-docs/images and core-docs/docs/images that don't seem to be in use. We should review those later.

qinezh committed:
Update .openpublishing.publish.config.json

chenkennt committed:
Initialize open publishing repository: https://github.com/dotnet/core-docs of branch live

chenkennt committed:
Merge branch 'master' into live

Commit:
Updated URLS, version numbers for .NET Core RTM

Commit:
Corrected URL for 'The Week in .NET'

mairaw committed:
fixed broken links, moved images and other minor fixes

mairaw committed:
fixed broken links, moved images and other minor fixes

mairaw committed:
mend

mairaw committed:
mend

Commit:
Removed double parens

enricosada committed:
add appveyor (#461)

* add appveyor

* dotnet core sdk preview1 is already installed in appveyor

* update docs

* improvements from feedback

brettcannon committed:
Escape the trailing `#` in title (#671)

* Escape the trailing `#` in title

https://docs.microsoft.com/en-us/dotnet/articles/fsharp/index is dropping the trailing `#`.

* Rename "Learn F#" to "F# Guide"

mairaw committed:
changing behavior of edit button

It should go to master branch instead of live

mairaw committed:
change edit button behavior

To go to master branch instead of live

tdykstra committed:
create c# 6 what's new stub (#672)

* create c# 6 what's new stub

* fix link

mairaw committed:
Fixing the edit button behavior

mairaw committed:
Fixing the edit button behavior

richlander committed:
Add telemetry topic (#684)

* Add telemetry topic

* Update text

* Line wrap embedded markdown

* Update per feedback

Commit:
trigger failed PR publish

tdykstra committed:
new getting started doc for core guide (#686)

* initial draft

tdykstra committed:
typo (#691)

jackbarclay committed:
Update using-on-macos.md

minor spelling corrections

Commit:
Merge pull request #673 from rpetrusha/welcome

Welcome

akoeplinger committed:
Fix extra dot in version number in managing-package-dependency-versions.md

jwmsft committed:
Fixes bold headers in Tutorial section (#693)

The bold headers in the Tutorials section had an extra space that causes them to render incorrectly.

Commit:
Update .net versions to 1.0.0, update donet-test-xunit to 1.0.0-* (#687)

* Update .net versions to 1.0.0, update donet-test-xunit to 1.0.0-*

* Change xunit to 2.2.0-beta2-build3300 and dotnet-test-xunit to 2.2.0-preview2-build1029

tdykstra committed:
move msbuild article to core/tutorials (#685)

tbertenshaw committed:
Update generics.md (#689)

grammar change, and fix the example code.

Commit:
Revisions to API portal

svick committed:
Corrected titles and headings (#690)

- one H1 per article
- removed - from metadata
- added missing # to titles

msetze01 committed:
Update index.md (#694)

* Update index.md

Fix typos

* Additional typographical edits

cartermp committed:
Merge pull request #692 from jackbarclay/patch-1

Update using-on-macos.md

Commit:
Added metadata

cartermp committed:
Merge pull request #695 from akoeplinger/patch-1

Fix extra dot in version number in managing-package-dependency-version.md

Commit:
Corrected metadata

ericstj committed:
Revert servicing (#700)

Commit:
Merge pull request #698 from rpetrusha/api

Revisions to API portal

mairaw committed:
Fixing a few typos on telemetry topic (#696)

* Fixing a few typos on telemetry topic

* updating title headers

richlander committed:
Add telemetry to TOC (#701)

JXPrime committed:
Updates docs for VS2015 info (#699)

Updates the information about determining the version of Visual Studio 2015.

richlander committed:
Merge remote-tracking branch 'origin/master' into live

Wind4 committed:
Fixed &lt; &gt; missing semi. (#703)

cartermp committed:
Update testing section to latest xUnit

cartermp committed:
Merge pull request #705 from cartermp/update-library-manual

Update testing section to latest xUnit

BillWagner commented:
LGTM. 
@mairaw Do you want to remove the do-not-merge label?  Should these go in now?

tdykstra committed:
move tutorials folder to csharp folder (#704)

mairaw commented:
Thanks @BillWagner. I can't. I still have to investigate why the image links are showing up as broken in the report.

dazfuller committed:
Corrected folder path for library output (#708)

mairaw committed:
Merge pull request #710 from dotnet/master

Merging master branch into live

tdykstra committed:
minor edits in libraries tutorial (#716)

jaredpsimpson committed:
Fixed C# tutorial links (#717)

mairaw committed:
Converted absolute URLs to API syntax (#713)

* Converted absolute URLs to API syntax

BillWagner committed:
Fix one project mis-match, and be consistent with static using (#719)

mairaw committed:
Delete duplicate file (#720)

mairaw committed:
fixing metadata issue (#718)

mairaw committed:
Fixing build warning (#721)

mairaw committed:
fix grammar issue (#709)

responding to feedback received on the site: 
Grammar nitpick: unique is a binary word, it cannot have a qualifier. Something either is unique or it's not, it cannot be "somewhat unique."

mairaw committed:
changed to ops notes (#702)

richlander committed:
Merge remote-tracking branch 'origin/master' into live

mairaw committed:
Broken link fixes and other issues (#724)

* fixes broken links, images and other minor things

* addressed feedback

* added  libraries to the TOC

* updated toc entry

* more fixes

bradygaster committed:
Update toc.yml

mairaw committed:
Merge pull request #732 from dotnet/master

Merging master branch into live

richlander committed:
Stevehoag using windows (#733)

* Update 'Using on Windows' topic

* Updated Using on Windows topic

Trying to fix build issue

* Updated Using on Windows topic

Now pointing to RTM versions

* Reapplied fix from svick

* Updated per PR feedback

* Triggering a build

mairaw committed:
fixing broken links (#734)

heyalistair committed:
fixing small mistake in the directory tree layout (#729)

Schema now follows instructions: "...create a 'golden' directory. Under that directory create `src` and `test` directories."

richlander committed:
Merge branch 'master' into live

bradygaster committed:
Merge pull request #731 from dotnet/bradygaster-toc

Update toc.yml to include link to docs home

cartermp committed:
Add Getting Started with F# on .NET Core (#728)

* Add Getting Started with .NET Core for F#

* Update summary at the top

* fix numbering

* feedback

* feedback

* feedback

* feedback

* feedback

srobati committed:
Update unit-testing-with-dotnet-test.md (#738)

richlander committed:
Change 'Getting Started' links to target the right page (#736)

* Change 'Getting Started' links to target the right page

* Update link per feedback

stefc committed:
version-suffix option was not marked as code (#744)

Commit:
Should be .NET Core instead of .NET Core 5 (#743)

mairaw committed:
Merge pull request #747 from dotnet/master

Merging master branch into live

richlander committed:
Clarify .NET Tools Telemetry (#749)

* Clarify .NET Tools Telemetry

* Update date

richlander committed:
Merge remote-tracking branch 'origin/live'

Xhanti committed:
Correction (#750)

Wording correction

svick committed:
Fixed link to open.xamarin.com (#752)

svick committed:
Fixed links in samples (#754)

svick committed:
Updated sample README (#753)

tdykstra committed:
change text block formatting (#755)

* change text block formatting

* change to text formatting

cartermp committed:
Unlink trimming doc (#767)

options committed:
insert step 4 in "Writing the consoling app" section (#745)

* insert step 4 in "Writing the consoling app" section

* Insert step 3 in "A mixed .NET Core library and .NET Framework application" section

* Replace platform version "1.0.0-rc2-3002702" with "1.0.0"

* remove "version" information in project.json

* Newtonsoft.Json 8.0.3 is not compatible for .net core 1.0. so changed version to 9.01

Duncanma committed:
Removing colons from the metadata as a test (#748)

* Removing colons from the metadata as a test

* fixed metadata

* testing different solution

* changed from escaping to quotes

BillWagner committed:
Update samples readme to match current  projects (#770)

* Update samples readme to match current  projects

* responding to review feedback.

sryabkov committed:
Update console-teleprompter.md (#771)

mairaw committed:
Merge pull request #773 from dotnet/master

Merging master branch into live

carlosmunozrodriguez committed:
Fixed some spelling mistakes (#775)

Fixed the following spellin mistakes:

orginal --> original
desgined --> designed
genreated --> generated
prioring --> prior

tugberkugurlu committed:
this should be 1.6? (#741)

richlander committed:
Merge remote-tracking branch 'origin/live'

Commit:
Starting the PJ schema changes (#448)

* Starting the PJ schema changes
* Update project-json-reference.md
A couple of fixes
* structural changes
* fix links for publish
* Add changed runtime options
* remove dead link
* Update project-json-reference.md
removed unnecessary headers, added some missing entries, changed boolean values to look consistent, filled out some to-dos.
* remove deprecated sections
* flesh out TODOs and examples
* adress feedback
* added more content
* fixed formatting
* removed dnx; updated versions
* some additional formatting fixes
* incorporated feedback/reorganized buildOptions section
* fixed second mappings
* additional changes
* fixed typos

mairaw committed:
fixing merge conflicts

mairaw commented:
I guess my rebase didn't work. :cry: 
I'll close and start this over.
